weave/urls.py

```diff
@@ -23,7 +25,7 @@ def project_root_url(entity_name: str, project_name: str) -> str:
 
 
 def project_path(entity_name: str, project_name: str) -> str:
-    return f"{project_root_url(entity_name, project_name)}"
+    return project_root_url(entity_name, project_name)
```

Comment:
@tssweeney - I'm not sure what the intention was for `project_path` vs. `project_root_url`. I don't see anywhere calling `project_path` so maybe we should just remove it?

Reply:
probably bad dev tracks and safe to remove